roles/core/conman/molecule/default/verify.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Verify
+  hosts: management1
+  tasks:
+
+    - name: Collect package facts
+      package_facts:
+        manager: auto
+
+    - name: Collect services facts
+      service_facts:
+
+    - name: Assert common packages are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - conman
+        - ipmitool
+      when:
+        - ansible_facts.os_family == "RedHat"
```

Comment:
Check for OpenSUSE as well.

Reply:
I wasn't able to run properly the role in opensuse 15.1.
Declaring an inventory variable `ansible_python_interpreter: /usr/bin/python3` does not solve this problem either.

ERROR:

```
TASK [Collect package facts] ***************************************************
Warning: 35m[WARNING]: Found "rpm" but Failed to import the required Python library (rpm)
on management1's Python /usr/bin/python. Please read the module documentation
and install it in the appropriate location. If the required library is
installed, but Ansible is using the wrong Python interpreter, please consult
the documentation on ansible_python_interpreter
fatal: [management1]: FAILED! => {"changed": false, "msg": "Could not detect a supported package manager from the following list: ['portage', 'pacman', 'rpm', 'pkg', 'apt'], or the required Python library is not installed. Check warnings for details."}
    
    PLAY RECAP *********************************************************************
    management1                : ok=1    changed=0    unreachable=0    failed=1
```

Reply:
I can't say, but last time this test ran against master, it was successful with OpenSUSE: https://github.com/bluebanquise/bluebanquise/actions/runs/329790788

Given that support of OpenSUSE is planned for the stack, I am not keen to remove any existing test.

Reply:
Tests added in latests commits